go.mod

```diff
@@ -104,3 +104,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => github.com/NDStrahilevitz/tracee/types v0.0.0-20230116133302-d841182726fe
```

Comment:
Remove and replace with a bump later.

Reply:
Yes, please remove this and make sure the git log is properly breaking lines and aligned (not sure it is, the web interface is weird). Tks